shard.yml

```diff
@@ -8,10 +8,16 @@ dependencies:
   any_hash:
     github: sija/any_hash.cr
 
+development_dependencies:
+  timecop:
+    github: TobiasGSmollett/timecop.cr
+  ameba:
+    github: veelenga/ameba
```

Comment:
Cool! You might want to add a version here. So it will not break your builds next time ameba is released